src/EVM/Format.hs

```diff
@@ -471,7 +471,7 @@ formatPartial = \case
   JumpIntoSymbolicCode pc idx -> "Encountered a jump into a potentially symbolic code region while executing initcode. pc: " <> pack (show pc) <> " jump dst: " <> pack (show idx)
 
 formatSomeExpr :: SomeExpr -> Text
-formatSomeExpr (SomeExpr e) = formatExpr e
```

Comment:
I added this change, because it's waaay nicer for the user, and we should be getting cleaner issues, too.